frappe/public/js/frappe/list/base_list.js

```diff
@@ -631,6 +643,7 @@ frappe.ui.FilterArea = class FilterArea {
 				default_value = null;
 			}
 			return {
+				doctype: this.list_view.meta.name,
```

Comment:
```suggestion
				doctype: this.list_view.doctype,
```
BaseList has doctype value set